lib/config/index.js

```diff
@@ -23,7 +23,7 @@ const packageConfig = {
   minimumCompatibleVersion: '0.5.0'
 }
 
-const configFilePath = path.join(appRootPath, 'config.json')
+const configFilePath = process.env.CMD_CONFIG_FILE || path.join(appRootPath, 'config.json')
```

Comment:
We should use also use `path.resolve` here. And then put it like this:

`const configFilePath = path.resolve(appRootPath, process.env.CMD_CONFIG_FILE ||  './config.json')`

Seems more intuitive to me. Even when I'm still not happy with having two locations where we resolve environment variables now .-.